modules/acquia_lift_publisher/src/Form/ContentPublishingForm.php

```diff
@@ -198,9 +229,9 @@ protected function setImageSelectorElement(array &$form, $entity_type_id, $bundl
     if (!empty($image_fields)) {
       $form['acquia_lift_preview_image'] = [
         '#type' => 'select',
-        '#title' => t("Select bundle's preview image field."),
+        '#title' => $this->t("Select bundle's preview image field."),
```

Comment:
Single quotes please.

Reply:
Please note the single quote inside the string.

Reply:
Missed it :)